src/sentry/models/commitauthor.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+
+from sentry.db.models import BoundedPositiveIntegerField, Model, sane_repr
+
+
+class CommitAuthor(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
+    name = models.CharField(max_length=128, null=True)
```

Comment:
Can we just make this a `TextField`? We don't exactly control this and I doubt git itself has limitations here.